extendedmodlog/eventmixin.py

```diff
@@ -355,22 +355,21 @@ async def _cached_message_delete(
             )
         if embed_links:
             embed = discord.Embed(
-                description=message.content,
+                description=f"**Message sent by {message.author.mention} deleted in {message_channel.mention}**\n{message.content}",
```

Comment:
This line won't be affected by people trying to use translation features. You mus encapsulate the english text in `_()` outside of the f-strings here.